pkg/cmd/gitprovider/list.go

```diff
@@ -42,12 +42,13 @@ var gitProviderListCmd = &cobra.Command{
 
 		for _, gitProvider := range gitProviders {
 			for _, supportedProvider := range supportedProviders {
-				if gitProvider.Id == supportedProvider.Id {
+				if gitProvider.ProviderId == supportedProvider.Id {
 					gitProviderViewList = append(gitProviderViewList,
 						gitprovider_view.GitProviderView{
-							Id:       gitProvider.Id,
```

Comment:
The mapping is wrong here.

`Id` should be `gitProvider.Id`
`ProviderId` should be `gitProvider.ProviderId`

Reply:
We shouldn't remove it. We need to map all the fields and then it's up to the view to render what's needed and what's not.

It's especially required so it gets outputted with `-f json`

Reply:
@the-johnwick `ProviderId` is still not set here from what I can see.

Reply:
@the-johnwick maybe you're not understanding me.

The object should be:

```golang
gitprovider_view.GitProviderView{
	Id:       gitProvider.Id,
        ProviderId: gitProvider.ProviderId,
	Name:     supportedProvider.Name,
	Username: gitProvider.Username,
	Alias:    gitProvider.Alias,
},
```

Reply:
@the-johnwick It's not correct again https://github.com/the-johnwick/daytona/blob/git-provider/pkg/cmd/gitprovider/list.go#L47.

Let's get these few things done and we'll merge this today.